x-pack/plugins/features/common/sub_feature.ts

```diff
@@ -68,6 +69,12 @@ export interface SubFeaturePrivilegeConfig
    * `read` is also included in `all` automatically.
    */
   includeIn: 'all' | 'read' | 'none';
+
+  /**
+   * The minimum supported license level for this sub-feature privilege.
+   * If no license level is supplied, then this privilege will be available for all valid licenses.
+   */
+  minimumLicense?: LicenseType;
```

Comment:
`If no license level is supplied, then this privilege will be available for all valid licenses`

I would expect the base feature's `minimumLicense`  to be applied instead?

Reply:
Good point, I'll update the comment to make this clearer